pkg/logs/status/status.go

```diff
@@ -56,6 +56,11 @@ func Clear() {
 
 // Get returns the status of the logs-agent computed on the fly.
 func Get() Status {
+	if builder == nil {
```

Comment:
I think this is a duplicate of https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/logs.go#L89 WDYT ?

Reply:
Still necessary for the warnings.